wordpress/wp-content/themes/cds-default/inc/template-functions.php

```diff
@@ -244,6 +244,56 @@ function language_switcher_output($languages)
 
     return $langs;
 }
+/*
+    TODO: Replace this hack function with a proper solution.
```

Comment:
Let's call this a workaround vs ` total hack` --- 

 total hack doesn't inspire confidence in the product if folks happen to be looking at the source code

Reply:
Fair enough. I wanted to impress the importance of finding a better long term solution.